src/App.tsx

```diff
@@ -1,10 +1,88 @@
-/* eslint-disable jsx-a11y/label-has-associated-control */
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useCallback, useEffect, useMemo, useState } from 'react';
+import classNames from 'classnames';
 import { UserWarning } from './UserWarning';
-import { USER_ID } from './api/todos';
+import { USER_ID, getTodos, uploadTodo } from './api/todos';
+import { Todo, emptyTodo } from './types/Todo';
+import { TodoStatus } from './types/TodoStatus';
+import { ErrorType } from './types/Errors';
+import { Header } from './components/Header';
+import { TodoList } from './components/TodoList';
+import { Footer } from './components/Footer';
+import { filterTodosByStatus } from './utils/helpers';
+import { TodoStatusRoutes } from './constants/TodoRoutes';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [newTodo, setNewTodo] = useState<Omit<Todo, 'id'>>(emptyTodo);
+  const [isLoading, setIsLoading] = useState<boolean>(false);
+  const [selectedTodoStatus, setSelectedTodoStatus] = useState<TodoStatus>(
+    TodoStatus.All,
+  );
+  const [errorMessage, setErrorMessage] = useState<ErrorType | ''>('');
```

Comment:
Add `DEFAULT` to `ErrorType`
```suggestion
  const [errorMessage, setErrorMessage] = useState<ErrorType>(ErrorType.DEFAULT);
```